doc/getting_started.rst

```diff
@@ -90,10 +90,18 @@ this folder must have the following things:
       presented in a rich literate programming fashion, without any output. To
       change the default pattern for execution and capture see
       :ref:`build_pattern`.
+    * The output that is captured while executing the ``.py`` files and
+      subsequently incorporated into the built documentation, can be finely
+      tuned. See :ref:`print_eval_repr`.
     * You can have sub-directories in your ``examples`` directory. These will be
       included as sub-sections of your gallery. They **must** contain their own
       ``README.txt`` or ``README.rst`` file as well.
 
+.. warning::
+
+   The variable name ``___`` (3 underscores) should never be used in your
```

Comment:
In principle eventually we can relax this by actually looking in the `ast` to see if `___` exists and if it does, use `____`, if it exists use `_____`, ..., etc. Eventually there will be enough underscores that one will be unused :)

But this can be another PR if we ever hit this case, unless you're really interested in digging through `ast` to figure it out

Reply:
At the moment...no. But I can look into it if someone complains :)

Reply:
I've thought about this and I don't think we could do it with `ast`. This converts the code in a code block into AST. It doesn't tell you what global variables exist - this would be found in `example_globals`. `example_globals` would cumulatively include all variables assigned in a `.py` file. This means that for the first code block we can safely check to see if `___` exists in `example_globals` but for subsequent code blocks in the same `.py` file, a variable `___` may exist simply because *we* assigned it previously.

For now, I can't think of a solution.

Reply:
That's fine, we can deal with it later if need be but YAGNI for now